hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/utils/ConversionUtils.java

```diff
@@ -173,6 +173,10 @@ public static long asExactLongValueOrZero(@NonNull final BigInteger value) {
      */
     public static com.esaulpaugh.headlong.abi.Address headlongAddressOf(@NonNull final AccountID accountID) {
         requireNonNull(accountID);
+        if (accountID.account().kind() == AccountID.AccountOneOfType.UNSET) {
```

Comment:
Why would this ever be possible? 🤔

Reply:
When the spenderId of the token is null, we then use the ZERO_ACCOUNT_ID, which has an alias == null and that's where we failed with NPE on line 182. I could move the check one level above, but am not sure there are other tests failing because of that.